src/CMakeLists.txt

```diff
@@ -214,6 +214,10 @@ ELSE()
   ENDIF()
 ENDIF()
 
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
As I understand it this requires CMake 3.1 (see above) however it might be possible to code around this {see this answer on [Stack Exchange](http://stackoverflow.com/a/31010221/4805858)}...